fe/fe-core/src/main/java/org/apache/doris/qe/SessionVariable.java

```diff
@@ -2245,6 +2247,9 @@ public boolean isEnableHboNonStrictMatchingMode() {
                     "When processing both \\n and \\r\\n as CSV line separators, should \\r be retained?"})
     public boolean keepCarriageReturn = false;
 
+    @VariableMgr.VarAttr(name = EXCHANGE_MULTI_BLOCKS_BYTE_SIZE,
+            description = {"enable exchange send multi blocks in one rpc, default is 256KB. negatives means disable"})
+    public int exchangeMultiBlocksByteSize = 256 * 1024;
```

Comment:
maybe we can add fuzzy test for exchangeMultiBlocksByteSize

Reply:
ok